dropwizard-core/src/main/java/io/dropwizard/cli/Cli.java

```diff
@@ -70,12 +71,12 @@ public boolean run(String... arguments) throws Exception {
                 parser.printVersion(stdOut);
             } else {
                 final Namespace namespace = parser.parseArgs(arguments);
-                if (namespace.get("is-help") == null) {
-                    final Command command = commands.get(namespace.getString(COMMAND_NAME_ATTR));
-                    command.run(bootstrap, namespace);
-                }
+                final Command command = commands.get(namespace.getString(COMMAND_NAME_ATTR));
+                command.run(bootstrap, namespace);
             }
             return true;
+        } catch (HelpScreenException e) {
```

Comment:
* Could we make a comment why `true` is returned? From the first glance it's not obvious that `HelpScreenException` is not an error. 

* Maybe it's better to name the variable `ignored` instead of `e`? I beleive that is a common name for ignored exceptions.